bin/stacks/analytics-stack.ts

```diff
@@ -525,54 +548,58 @@ export class AnalyticsStack extends cdk.NestedStack {
     }
 
     /* log processor alarms */
-    [quoteProcessorLambda, fillEventProcessorLambda, postOrderProcessorLambda, botOrderEventsProcessorLambda].forEach(
-      (lambda) => {
-        const successRateSev2Name = `${lambda.node.id}-SEV2-SuccessRate`;
-        const successRateSev3Name = `${lambda.node.id}-SEV3-SuccessRate`;
-
-        const errors = lambda.metricErrors({
-          period: cdk.Duration.minutes(5),
-          statistic: cdk.aws_cloudwatch.Stats.SUM,
-          label: `${lambda.node.id} Errors`,
-        });
-
-        const invocations = lambda.metricInvocations({
-          period: cdk.Duration.minutes(5),
-          statistic: cdk.aws_cloudwatch.Stats.SUM,
-          label: `${lambda.node.id} Invocations`,
-        });
-
-        const successRate = new cdk.aws_cloudwatch.MathExpression({
-          expression: '100 - 100 * errors / MAX([errors, invocations])',
-          usingMetrics: {
-            errors,
-            invocations,
-          },
-          label: `${lambda.node.id} Success Rate`,
-        });
-
-        const successRateSev2 = new cdk.aws_cloudwatch.Alarm(this, successRateSev2Name, {
-          metric: successRate,
-          threshold: 60,
-          evaluationPeriods: 1,
-          comparisonOperator: cdk.aws_cloudwatch.ComparisonOperator.LESS_THAN_OR_EQUAL_TO_THRESHOLD,
-          actionsEnabled: true,
-        });
-
-        const successRateSev3 = new cdk.aws_cloudwatch.Alarm(this, successRateSev3Name, {
-          metric: successRate,
-          threshold: 90,
-          evaluationPeriods: 1,
-          comparisonOperator: cdk.aws_cloudwatch.ComparisonOperator.LESS_THAN_OR_EQUAL_TO_THRESHOLD,
-          actionsEnabled: true,
-        });
-
-        if (chatBotTopic) {
-          successRateSev2.addAlarmAction(new cdk.aws_cloudwatch_actions.SnsAction(chatBotTopic));
-          successRateSev3.addAlarmAction(new cdk.aws_cloudwatch_actions.SnsAction(chatBotTopic));
-        }
+    [
+      quoteProcessorLambda,
+      fillEventProcessorLambda,
+      postOrderProcessorLambda,
+      botOrderEventsProcessorLambda,
+      activeOrderProcessorLambda,
+    ].forEach((lambda) => {
+      const successRateSev2Name = `${lambda.node.id}-SEV2-SuccessRate`;
+      const successRateSev3Name = `${lambda.node.id}-SEV3-SuccessRate`;
+
+      const errors = lambda.metricErrors({
+        period: cdk.Duration.minutes(5),
+        statistic: cdk.aws_cloudwatch.Stats.SUM,
+        label: `${lambda.node.id} Errors`,
```

Comment:
do you know what this internal id ends up being? worried that it will be some random bytes string